integration/terraform/ec2/linux/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_instance" "integration-test" {
+  ami                    = data.aws_ami.latest.id
+  instance_type          = var.ec2_instance_type
+  key_name               = var.key_name
+  iam_instance_profile   = var.iam_instance_profile
+  vpc_security_group_ids = var.vpc_security_group_ids
+  provisioner "remote-exec" {
+    inline = [
+      "cloud-init status --wait",
+      "echo clone, build, and install agent",
+      "git clone ${var.github_repo}",
+      "cd amazon-cloudwatch-agent",
+      "git reset --hard ${var.github_sha}",
+      "make clean build ${var.package}",
```

Comment:
On the one hand, I feel like the usage of the `package` var is obvious, but on the other hand, it's not - even though I read the action first, when I got to this, I thought that `package` referred to like, the RPM or package name, and not the Makefile action for packaging. I can't think of a much better name for it though. I'll have to think about that one.